src/OpenLoco/Math/Trigonometry.hpp

```diff
@@ -0,0 +1,116 @@
+#pragma once
+#include "Vector.hpp"
+#include <cstdint>
+
+namespace OpenLoco
+{
+    enum class Pitch : uint8_t;
+}
+
+namespace OpenLoco::Math::Trigonometry
+{
+    // 0x00503B6A
+    // ROUND(COS((32/64+(L1/64))*(2*PI()))*256,0), ROUND(SIN(((L1/64))*(2*PI())) * 256,0)
+    // Where L1 represents an incrementing column 0 - 63
+    // Note: Must be at least 32bit to ensure all users do not overflow
+    static constexpr Vector::TVector2<int32_t, 1> factorXY503B6A[64] = {
```

Comment:
We're so close to a good cleanup here. I just think we should try and name these. Do these represent yaw and roll?

Reply:
`yawToDirectionVector`?

Reply:
Sounds good to me